### PR TITLE
Release of GeoNetwork 4.2.0.

### PR DESCRIPTION
**Pull request by fxprunayre**

### CODE REVIEW

**Comment by juanluisrp** [comment]
@fxprunayre can you add a warning in https://github.com/docker-library/docs/blob/master/geonetwork/content.md, for example at the start of the _How to use this image_ section about the 4.2.0 and the H2 update procedure?

File: 4.2.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    rm -rf /var/lib/apt/lists/* && \
+    mkdir -p /${DATA_DIR} && \
+    chown -R jetty:jetty ${DATA_DIR} && \
+    mkdir -p /var/lib/jetty/webapps/geonetwork && \
+    chown -R jetty:jetty /var/lib/jetty/webapps/geonetwork
+
+USER jetty
+ENV GN_FILE geonetwork.war
+ENV GN_VERSION 4.2.0
+ENV GN_DOWNLOAD_MD5 3351b899162b9260459c6c76d1d8f446
```
Comment (juanluisrp) [inline 4.2.0/Dockerfile:24]: I'm getting this error when trying to build the image from `4.2.0`:

```
md5sum: WARNING: 1 computed checksum did NOT match.
```

MD5 defined in the Dockerfile is not valid. ` GN_DOWNLOAD_MD5` is set to `3351b899162b9260459c6c76d1d8f446` but it should be `5300f81f24f1585d969e667d922e8e8e`.

**Review by juanluisrp — CHANGES_REQUESTED** [review]

File: 4.2.0/Dockerfile (hunk 1 above)
Comment (juanluisrp) [reply 4.2.0/Dockerfile]: Updated.

**Review by juanluisrp — APPROVED** [review]

**Comment by fxprunayre** [comment]
Thanks @juanluisrp

**Pull request merged**